database/src/lib.rs

```diff
@@ -852,6 +863,18 @@ impl BenchmarkRequestType {
             BenchmarkRequestType::Release { tag: _ } => "release",
         }
     }
+
+    /// For getting the priority of the request type;
+    /// - Release
+    /// - Master
+    /// - Try
+    pub fn rank(&self) -> u8 {
```

Comment:
The rank functions should be in `site`, not `database`, IMO. They return ordering that is specific to some logic in `site`, elsewhere we might order these things differently.

Reply:
:+1: Deleted and made them functions in `job_queue.rs`.